src/styles/components/_list-group.scss

```diff
@@ -0,0 +1,29 @@
+@import '../variables';
+
+$border-radius-list-group: 3px;
```

Comment:
It's the same with global border-radius
We can move this variable to _variable.scss

Reply:
Fixed